harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -826,9 +1011,10 @@ def _save(self, path: pathlib.Path) -> None:
 
         torch.save(checkpoint, str(path.joinpath("state_dict.pth")))
 
-        if self.wlsq is not None:
+        if self.save_state is not None:
+            self.save_state.steps_completed = self.steps_completed
             with path.joinpath("workload_sequencer.pkl").open("wb") as f:
```

Comment:
also, how will the transition from wlsq to new pytorch training loop work?

Will paused experiments be able to continue after this upgrade?

Reply:
yes because existing implementation relies on steps_completed, which is always in batches. so loading from a previous save state should resume right where you left off.